libs/wingsdk/src/target-sim/util.ts

```diff
@@ -61,3 +61,29 @@ export function makeSimulatorJsClient(filename: string, resource: Resource) {
       })("${env}")`
   );
 }
+
+// helper function to convert duration to a cron string
+// maybe this belongs in a util library but for now it's here
+export function convertDurationToCronExpression(dur: Duration): string {
+  if (dur.minutes % 1 !== 0) {
+    // our cron expression format is [minute] [hour] [day] [month] [year]
+    throw new Error("Cron expressions with second precision are not supported");
+  }
+
+  const totalInMinutes = Math.floor(dur.minutes);
+
+  const h = Math.floor(totalInMinutes / 60);
+  const m = totalInMinutes % 60;
+
+  const minute = m != 0 ? `*/${m}` : "*";
+  const hour = h != 0 ? `*/${h}` : "*";
+  // TODO: Support longer durations once we implement https://github.com/winglang/wing/issues/2243
```

Comment:
I think it could also be powerful to support sub-minute durations as a future capability (see [here](https://constructs.dev/packages/cdk-lambda-subminute/v/2.0.109?lang=typescript) for an example implemented in CDK). Let's create an issue if we don't have one already

Reply:
I did not see an issue so I created this bad boy: https://github.com/winglang/wing/issues/2350